### PR TITLE
test: Prep env for custom NSS build

### PR DESCRIPTION
**Pull request by larseggert**

### CODE REVIEW

File: test/test.sh [hunk 1]
```diff
@@ -23,6 +23,11 @@ else
         iface=lo
 fi
 
+if [ "$NSS_DIR" ] && [ "$NSS_TARGET" ]; then
+        export LD_LIBRARY_PATH=$NSS_DIR/../dist/$NSS_TARGET/lib
+        export DYLD_FALLBACK_LIBRARY_PATH=$LD_LIBRARY_PATH
```
Comment (martinthomson) [inline test/test.sh:28]:
```suggestion
        export LD_LIBRARY_PATH="$NSS_DIR/../dist/$NSS_TARGET/lib"
        export DYLD_FALLBACK_LIBRARY_PATH="$LD_LIBRARY_PATH"
```

File: test/test.sh [hunk 2]
```diff
@@ -23,6 +23,11 @@ else
         iface=lo
 fi
 
+if [ "$NSS_DIR" ] && [ "$NSS_TARGET" ]; then
```
Comment (martinthomson) [inline test/test.sh:26]: Huh, TIL.

 I would have used `[[ -n "$NSS_DIR" -a -n "$NSS_TARGET" ]]`.  I didn't even know that you could do `[ "$foo" ]`.  Is that even robust against the variable having a value of something like "!"?

**Review by martinthomson — APPROVED** [review]

File: test/test.sh (hunk 2 above)
Comment (larseggert) [reply test/test.sh:26]: https://www.shellcheck.net/wiki/SC2108

Comment (larseggert) [reply test/test.sh:26]: I love shellcheck

Comment (larseggert) [reply test/test.sh:26]: `[` is `test`.

Comment (martinthomson) [reply test/test.sh:26]: Yeah, bash manual recommends `[[` over `[`/`test`.

The nice thing about shellcheck is that it is rarely wrong.  That's something to be treasured.

**Pull request merged**